### ISSUE TITLE
Sequential run not working (queued execution cancelled automatically)

### ISSUE THREAD

**Issue by pedrovgp** [jupyter/notebook#5098]
If I run a bunch of cells in sequence, only the first runs. All others get a "finished" timestamp equal to the first cell's timestamp, but none was really executed. In the picture below, all cells were queued almost at the same time.
![image](https://user-images.githubusercontent.com/2333868/70087960-0ba83d00-15f4-11ea-848c-af7e110b29d6.png)

Another simple step-by-step example:
1. I ran the first cell, then the second after the first finished executing.
![image](https://user-images.githubusercontent.com/2333868/70087507-3a71e380-15f3-11ea-8453-9a8d3a195701.png)
2. I modified b to be equal 23, then ran both cells in a sequence. The second cell's execution was queued, because the first execution had not finished. At the end, I got this:
![image](https://user-images.githubusercontent.com/2333868/70087681-80c74280-15f3-11ea-97b5-d9800867ce85.png)
3. I checked to see if b was modified, even though the cell did not show any output. But it was not:
![image](https://user-images.githubusercontent.com/2333868/70087732-a18f9800-15f3-11ea-8a5d-9b1db9cf96fa.png)

**Comment by Mateil04** [comment]
Same issue here. Any solution so far? Thanks!

**Comment by pedrovgp** [comment]
Not really, I ended up going back to databricks, this is kind of a deal breaker.

**Comment by jasongrout** [comment]
Does your server log show any errors? Do you see any errors in the javascript console in the web browser?

If not, can you reproduce the issue in a new, clean environment with a fresh installation of just the notebook?
